docs/modules/game-object-creation/custom-usage.md

```diff
@@ -0,0 +1,52 @@
+# Custom spawning logic
+
+The Game Object Creation Feature Module offers you a simple way to customize how GameObjects are created. You must:
+
+1. Implement a class which inherits from the `IEntityGameObjectCreator` interface.
+2. Tell the GDK to use this implementation.
+
+## Implementing the `IEntityGameObjectCreator` interface
+
+This interface contains two methods which each have invariants that must be respected.
```

Comment:
should we mention which invariants they have?

Reply:
Its covered within the sections, implicitly

Reply:
I'd rather not duplicate the information and keep it local to the section where it most makes sense

Reply:
in that case we shouldn't mention it at all. It's a slightly confusing transition from "there are invariants" to "here is an example implementation"

Reply:
Yeah sure can omit it instead